apps/ledger-live-mobile/src/components/SelectDevice2/index.tsx

```diff
@@ -243,14 +259,25 @@ export default function SelectDevice({
   const onAddNewPress = useCallback(() => setIsAddNewDrawerOpen(true), []);
 
   const openBlePairingFlow = useCallback(() => {
+    // When starting the ble pairing flow, the tab bottom bar is not displayed
+    dispatch(updateMainNavigatorVisibility(false));
     setIsAddNewDrawerOpen(false);
     setIsPairingDevices(true);
-  }, []);
+  }, [dispatch]);
+
+  useEffect(() => {
+    return () => {
+      // Makes sure that on go back/unmount the visibility of the bottom tab bar is reset
```

Comment:
maybe you need to do this too when the component is not in a focused screen too? 🤔

Reply:
a `useEffect` with `const isFocused = useIsFocused()` as a dependency

Reply:
I agree - Adding

```javascript
  // Makes sure that when loosing (screen) focus, the visibility of the bottom tab bar is reset
  useEffect(() => {
    return () => {
      // Just before cleaning, the associated screen had focus
      if (isFocused) {
        dispatch(updateMainNavigatorVisibility(true));
      }
    };
  }, [dispatch, isFocused]);
```